_tips/writing.id.md

```diff
@@ -0,0 +1,470 @@
+    <div>
+      <svg version="1.1" width="462" height="27">
+        <g transform="scale(1.2)">
+          <path class="banner" d="M 0 9.75 L 0 14.8125 L 0 21.625 L 384.5 21.625 L 384.5 14.8125 L 384.5 9.75 L 384.5 3.25 C 384.5 1.59314575 383.15685 .25 381.5 .25 L 3 .25 C 1.34314575 .25 0 1.59314575 0 3.25 Z" fill="#f5f5f5"/>
+          <text transform="translate(10 15)">Berita terbaru &bull; Space Teddy Inc.</text>
+          <g transform="translate(333 4)">
+            <rect x="0" y="9.5" width="10" height="1.5"/>
+            <path d="M 24 10 L 17 10 L 17 4 L 24 4 Z M 16 11 L 25 11 L 25 3 L 16 3 Z"/>
+            <path d="M 41 3 L 40 2 L 36 6 L 32 2 L 31 3 L 35 7 L 31 11 L 32 12 L 36 8 L 40 12 L 41 11 L 37 7 Z"/>
+          </g>
+        </g>
+      </svg>
+    </div>
+  </figure>
+  <figure>
+    <figcaption>{% include_cached icon.html name="check-circle" label="Oke" %} Nama halaman termasuk langkah pada suatu proses</figcaption>
+    <div>
+      <svg version="1.1" width="462" height="27">
+        <g transform="scale(1.2)">
+          <path class="banner" d="M 0 9.75 L 0 14.8125 L 0 21.625 L 384.5 21.625 L 384.5 14.8125 L 384.5 9.75 L 384.5 3.25 C 384.5 1.59314575 383.15685 .25 381.5 .25 L 3 .25 C 1.34314575 .25 0 1.59314575 0 3.25 Z" fill="#f5f5f5"/>
+          <text transform="translate(10 15)">Beli Beruangmu (Langkah 1 dari 3) &bull; Space Teddy Inc.</text>
+          <g transform="translate(333 4)">
+            <rect x="0" y="9.5" width="10" height="1.5"/>
+            <path d="M 24 10 L 17 10 L 17 4 L 24 4 Z M 16 11 L 25 11 L 25 3 L 16 3 Z"/>
+            <path d="M 41 3 L 40 2 L 36 6 L 32 2 L 31 3 L 35 7 L 31 11 L 32 12 L 36 8 L 40 12 L 41 11 L 37 7 Z"/>
+          </g>
+        </g>
+      </svg>
+    </div>
+  </figure>
+</div>
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="Informasi Lebih Lanjut" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [Halaman diberi Titel 2.4.2](/WAI/WCAG21/quickref/#page-titled) ([Memahami 2.4.2](/WAI/WCAG21/Understanding/page-titled))
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## Gunakan judul untuk menyampaikan makna dan struktur
+
+Gunakan judul pendek untuk mengelompokkan paragraf terkait dan menerangkan bagian-bagiannya dengan jelas. Judul yang baik menggambarkan garis besar isinya.
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="Contoh: Gunakan judul untuk mengelompokkan konten" class="example" %}
+{:/}
+
+<div class="heading-structure two-column">
+  <figure>
+    <figcaption>{% include_cached icon.html name="ex-circle" label="Salah" %} Kekurangan judul</figcaption>
+    <div>
+      {% include_cached img.html src="headings-poor.png" alt="Contoh tidak ada judul, lihat di bawah untuk contoh mendetail" %}
+    </div>
+{% include_cached excol.html type="start" id="no-headings" %}
+Lihat contoh *inline*
```

Comment:
@frirasyidi If "inline" is an anglicism, would you wrap it with `<span lang"en"></span>`?

Reply:
@remibetin true! I missed that.
Fixed on the next commit.